.circleci/config.yml

```diff
@@ -39,6 +38,18 @@ jobs:
         command: gem update --system && gem install bundler && bundle update
     - run: bundle exec rake circleci:build
 
+  post-build:
+    docker:
+    - image: circleci/ruby:2.5.1-stretch
+    working_directory: ~/GoogleCloudPlatform/google-cloud-ruby
+    shell: /bin/bash --login -eo pipefail
+    steps:
+    - checkout
+    - run:
+        name: Install bundle
+        command: gem update --system && gem install bundler && bundle update
+    - run: bundle exec rake circleci:post
+
```

Comment:
Did you intend to omit coveralls here?

Reply:
`post` now calls `coveralls` as well as the doc build tasks.